Apps/Sandcastle/gallery/ScriptInInfoBox.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Click on the red polygon in order to see the script working.">
+    <meta name="cesium-sandcastle-labels" content="Tutorials">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+viewer.infoBox.allowScripts = true; // now the info box can have scripts
+    var redDesc = '<script>alert("x");<\/script>';
+var redPolygon = viewer.entities.add({
+    name : 'Red polygon on surface',
+    description: redDesc,
+    polygon : {
+        hierarchy : Cesium.Cartesian3.fromDegreesArray([-115.0, 37.0,
+                                                        -115.0, 32.0,
+                                                        -107.0, 33.0,
+                                                        -102.0, 31.0,
+                                                        -102.0, 35.0]),
+        material : Cesium.Color.RED
+    }
+});
+
+var greenPolygon = viewer.entities.add({
```

Comment:
99% of the code in this example can be removed.  It was also not immediately clear to me what I was supposed to do when running it.  Perhaps just use a single Label that says "Click me!" and then executes the script when the user clicks.

Reply:
Done